client/src/services/VideoService.js

```diff
@@ -20,6 +20,9 @@ const videoStorageBucket = new AWS.S3({
   region: REGION,
 });
 
+// Max file size = 20 MB
+const MAX_FILE_SIZE = 20 * 1024 * 1024;
```

Comment:
:memo: Adding a max file size to s3 video uploads for now. If you download YT videos in 720p they're not too big. Doing this as to not exceed S3 free tier limit.